lib/src/components/VernaForm/index.spec.tsx

```diff
@@ -19,29 +21,51 @@ describe('VernaForm', () => {
     await userEvent.click(_.last(buttons)!);
   }
 
-  it('should render a basic form', async () => {
+  const Wrapper = ({
+    isEditor,
+    selector,
+    uiSchema,
+    widgets,
+  }: {
+    selector?: string;
+    isEditor?: boolean;
+    uiSchema?: UiSchema;
+    widgets?: WidgetsType;
+  }) => {
     const translations = translationsFactory();
-
-    render(
-      <VernaProvider
-        isEditor
-        defaultSchema={schemaFactory()}
-        defaultUiSchema={uiSchemaFactory()}
-        defaultWidgets={widgetsFactory()}
-        locale="en"
-        translations={translations}
-      >
-        <VernaForm />
-      </VernaProvider>,
+    return (
+      <Suspense fallback="Loading...">
```

Comment:
There aren't any other way to do this to avoid experimental features?
https://fr.reactjs.org/docs/concurrent-mode-suspense.html